keycloak-apikey/src/main/java/eu/europeana/keycloak/validation/service/ListApiKeysService.java

```diff
@@ -0,0 +1,107 @@
+package eu.europeana.keycloak.validation.service;
+
+import eu.europeana.keycloak.validation.datamodel.Apikey;
+import eu.europeana.keycloak.validation.util.Constants;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.Date;
+import java.util.List;
+import org.apache.commons.lang3.StringUtils;
+import org.jboss.logging.Logger;
+import org.keycloak.models.ClientModel;
+import org.keycloak.models.KeycloakSession;
+import org.keycloak.models.RoleModel;
+import org.keycloak.models.UserModel;
+
+public class ListApiKeysService {
+  private static final Logger LOG  = Logger.getLogger(ListApiKeysService.class);
+  private final KeycloakSession session;
+
+  public ListApiKeysService(KeycloakSession session) {
+    this.session = session;
```

Comment:
this is not used anywhere but i assume this required to attach a session for the user while making a request